### PR TITLE
Adoption: fix stopping nova services from tripleo

### PR DESCRIPTION
**Pull request by bogdando**
Match the target container by a strict name.
Filter unrelated nova_virtlogd_wrapper container
and make the nova_virtlogd stopping condition triggering.

Move the stopping logic for nova_virtlogd_wrapper and nova_virtlogd
into the libvirt role as we need to stop these *before* we start
deploying de-containerized libvirt. While the nova role is called
after the libvirt role, so stopping the tripleo libvirt related
happens too late (libvirt has already failed).

Also stop other Nova control plane services, in case of all-in-one
overcloud adoption.

JIRA [OSPRH-2302](https://issues.redhat.com/browse/OSPRH-2302)

### CODE REVIEW

**Review by mrkisaolamb — APPROVED** [review]
lgtm

**Comment by bogdando** [comment]
/hold Now I realised that this task should belong to libvirt
we deploy nova after libvirt, and virtlogd makes the latter failing, before the former can fix it

**Comment by bogdando** [comment]
recheck unrelated

File: roles/edpm_nova/defaults/main.yml
Comment (SeanMooney) [inline roles/edpm_nova/defaults/main.yml:39]: hum ok yes if you were adopting form standalone this would be needed.

**Review by SeanMooney — COMMENTED** [review]
recheck 

this looks good. 
it fixes the responsibilities between the roles
the main feedback is there is no molecule testing for the adoption behavior.

i know gibi was looking at adding an adoption seneario so perhaps we could cover that there but ideally we would use tasks in prepare to create a fake systems service to emulate trileo deployed services.

is this something you plan to do in this pr.

File: roles/edpm_nova/defaults/main.yml
Comment (bogdando) [reply roles/edpm_nova/defaults/main.yml:39]: there is also all-in-one overcloud managed with Director supported in OSP (which is NOT standalone tripleo)

**Comment by bogdando** [comment]
recheck edpm-ovn

**Comment by mrkisaolamb** [comment]
recheck

**Review by SeanMooney — APPROVED** [review]
very well, lets follow up with the molecule tests then and proceed with this to unblock adoption work

**Pull request merged**
